src/approval/ApprovalController.test.js

```diff
@@ -118,7 +118,9 @@ describe('ApprovalController: Input Validation', () => {
     let approvalController;
 
     beforeEach(() => {
-      approvalController = new ApprovalController({ ...defaultConfig });
+      approvalController = new ApprovalController(
+        Object.assign({}, defaultConfig),
```

Comment:
Hm. This is unexpected.

Reply:
This was a bit lazy on my part. We can allow >ES2017 here since it's just unit tests. Just by default our config uses ES2017, which is appropriate.

Reply:
Updated to use ES2018 for JavaScript files, so these Object.assign's have been removed.